.castor/utils.php

```diff
@@ -101,6 +101,28 @@ function create_default_context(): Context
     return new Context($data, pty: 'dev' === $data['env']);
 }
 
+function docker_exit_code(
```

Comment:
Maybe `docker_compose_run_exit_code` is a more appropriate name?

Reply:
this is **very** long :(